Woof/WoofTests/ViewModel/LoginViewModelTests/LoginViewModelSetUserRoleTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class LoginViewModelSetUserRoleTests: XCTestCase {
+    private var userPreferencesStorage = KeyValueStorage.testPreferencesStorage
+    private let testLoginViewModel = LoginViewModel()
+
+    override func setUp() {
+        super.setUp()
+        userPreferencesStorage.deleteData(for: KeyValueStorage.Key.userPreferences)
+    }
+
+    func testMethodExistsInAPI() {
+        _ = testLoginViewModel.set(.sitter)
+    }
+
+    func testReturnsTrueWhenSetEveryPossibleRole() {
+        Role.allCases.forEach {
```

Comment:
Do we expect that the role `.none` might be passed?

Reply:
no